src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/JudicialDecisionHandler.java

```diff
@@ -495,9 +500,11 @@ private CaseData.CaseDataBuilder getSharedData(CallbackParams callbackParams) {
     }
 
     private CallbackResponse setJudgeBusinessProcess(CallbackParams callbackParams) {
+        String eventIt = callbackParams.getRequest().getEventId();
```

Comment:
Could we name this variable in relation to what it holds?

Reply:
Done. typo fiexed.